app/page.tsx

```diff
@@ -66,13 +68,58 @@ function AboutUsOverview() {
   );
 }
 
+function SponsorsPreview() {
+  return (
+    <Box mx='auto' textAlign='center' width='100%' mb={10} justifyContent='center'>
```

Comment:
Just cleaning up the code, it looks like you can remove the textAlign and width props